src/components/simple-table/SimpleTable.module.scss

```diff
@@ -2,7 +2,6 @@
 
 .tableContainer {
   width: 100%;
-  overflow: auto;
```

Comment:
I probably added this when I was doing the sticky header. If it still works in dashboard then that's fine.